sw/airborne/modules/computer_vision/opticflow/linear_flow_fit.c

```diff
@@ -0,0 +1,430 @@
+/*
+ * Copyright (C) 2014 Hann Woei Ho
+ *		      Guido de Croon
+ *					  
+ * Code based on the article:
```

Comment:
This info would make more sense in the detailed doxygen description so it will show up in the generated docs...